### ISSUE TITLE
Move conversion.conf from TB project?

### ISSUE THREAD

**Issue by darrenleeweber** [sul-dlss/ld4p-tracer-bullet-scripts#22]
See https://github.com/sul-dlss/ld4p-tracer-bullets/blob/master/conversiontracerbullet/src/main/resources/conversion.conf - settings for LOC converter belong here?

**Comment by darrenleeweber** [comment]
The `conversion.conf` file contains settings for the LOC converter, i.e.
```
BASEURI="http://linked-data.stanford.edu/"
CREATE_BNODE=false
CLEANUP=true
```

These might belong in the bash function defined to run the LOC converter.

**Comment by darrenleeweber** [comment]
- By default, the LOC converter sets `usebnodes=false`
  - so the `CREATE_BNODES` option is not required
- This scripts project now sets and uses `LD4P_BASEURI` in the `ld4p_configure.sh`
  - https://github.com/sul-dlss/ld4p-tracer-bullet-scripts/blob/master/ld4p_configure.sh#L11
  - so the `BASEURI` option is not required
- There is no option on the LOC converter for `CLEANUP`
  - so the `CLEANUP` option is not required
  - in fact, this is no longer used on the parent java project, see https://github.com/sul-dlss/ld4p-tracer-bullets/blob/3227586cdddb557718c746a3a77fcc9b310d95c2/conversiontracerbullet/src/main/java/org/stanford/ConvertToBibframe.java#L35

In conclusion, these settings are either already set in this TB-scripts project or they are the LOC converter default or they are deprecated.
